doc/operator/deploy-operator.md

```diff
@@ -18,7 +18,7 @@ In order to ease the installation of Observatorium, an operator is available.
 
 #### Create Namespaces
 ```shell script
-kubectl create namespace minio
```

Comment:
since this is not actually a requirement for deploying the operator, could we move it like 5 lines down where we describe the creation of minio for testing purposes?